server/test/services/program/ProgramServiceTest.java

```diff
@@ -2187,192 +2185,6 @@ public void newProgramFromExisting() throws Exception {
-            .setLocalizedSummaryImageDescription("German Image Description")
-            .setStatuses(
-                ImmutableList.of(
-                    LocalizationUpdate.StatusUpdate.builder()
-                        .setStatusKeyToUpdate(STATUS_WITH_EMAIL_ENGLISH_NAME)
-                        .setLocalizedStatusText(Optional.of("german-status-with-email"))
-                        .setLocalizedEmailBody(Optional.of("german email body"))
-                        .build(),
-                    LocalizationUpdate.StatusUpdate.builder()
-                        .setStatusKeyToUpdate(STATUS_WITH_NO_EMAIL_ENGLISH_NAME)
-                        .setLocalizedStatusText(Optional.of("german-status-with-no-email"))
-                        .build()))
-            .setScreens(ImmutableList.of())
-            .build();
-    ErrorAnd<ProgramDefinition, CiviFormError> result =
-        ps.updateLocalization(program.id, Locale.GERMAN, updateData);
-
-    assertThat(result.isError()).isFalse();
-    ProgramDefinition definition = result.getResult();
-    assertThat(definition.localizedName().get(Locale.GERMAN)).isEqualTo("German Name");
-    assertThat(definition.localizedDescription().get(Locale.GERMAN))
-        .isEqualTo("German Description");
-    assertThat(definition.localizedSummaryImageDescription().isPresent()).isTrue();
-    assertThat(definition.localizedSummaryImageDescription().get().get(Locale.GERMAN))
-        .isEqualTo("German Image Description");
-    ImmutableList<StatusDefinitions.Status> expectedStatuses =
-        ImmutableList.of(
-            StatusDefinitions.Status.builder()
-                .setStatusText(STATUS_WITH_EMAIL.statusText())
-                .setLocalizedStatusText(
-                    STATUS_WITH_EMAIL
-                        .localizedStatusText()
-                        .updateTranslation(Locale.GERMAN, "german-status-with-email"))
-                .setLocalizedEmailBodyText(
-                    Optional.of(
-                        STATUS_WITH_EMAIL
-                            .localizedEmailBodyText()
-                            .get()
-                            .updateTranslation(Locale.GERMAN, "german email body")))
-                .build(),
-            StatusDefinitions.Status.builder()
-                .setStatusText(STATUS_WITH_NO_EMAIL.statusText())
-                .setLocalizedStatusText(
-                    STATUS_WITH_NO_EMAIL
-                        .localizedStatusText()
-                        .updateTranslation(Locale.GERMAN, "german-status-with-no-email"))
-                .build());
-    assertThat(definition.statusDefinitions().getStatuses()).isEqualTo(expectedStatuses);
-    assertThat(
-            applicationStatusesRepo
-                .lookupActiveStatusDefinitions(definition.adminName())
-                .getStatuses())
-        .isEqualTo(expectedStatuses);
-  }
-
-  @Test
-  public void updateLocalizations_updatesExistingLocale() throws Exception {
```

Comment:
We shouldn't move this whole test into the `StatusService`, just the parts that are testing statuses. We can leave the rest here.

ditto for the other tests that we moved in their entirety.

Reply:
split these tests into programServiceTest and StatusServiceTest